### PR TITLE
Fix: Reduce Prop Drilling: Utils 

### PR DESCRIPTION
**Pull request by louisdevzz**

### CODE REVIEW

File: apps/potlock/widget/Cart/BreakdownSummary.jsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
-const { basisPointsToPercent, referrerId, amountNear, bypassProtocolFee, recipientId } = props;
-const { DONATION_CONTRACT_ID, SUPPORTED_FTS } = VM.require("potlock.near/widget/constants") || {
+const { referrerId, amountNear, bypassProtocolFee, recipientId, SUPPORTED_FTS } = props;
```
[the remark this answers is not in the document]
Comment (louisdevzz) [reply apps/potlock/widget/Cart/BreakdownSummary.jsx:1]: It doesn't work when I put it in the constant I tried it and it doesn't work

File: apps/potlock/widget/Index.jsx [hunk 2]
```diff
@@ -391,48 +291,50 @@ const props = {
...
-    const allowed = userRoles.some(({ permissions }) => {
-      return (
-        permissions.includes(`${kind}:${action}`) ||
-        permissions.includes(`${kind}:*`) ||
-        permissions.includes(`*:${action}`) ||
-        permissions.includes("*:*")
-      );
-    });
-    return allowed;
+  SUPPORTED_FTS: {
```
[the remark this answers is not in the document]
Comment (louisdevzz) [reply apps/potlock/widget/Index.jsx:297]: It didn't work so I put it back

File: apps/potlock/widget/Index.jsx [hunk 3]
```diff
@@ -391,48 +291,50 @@ const props = {
...
   },
   openDonateToProjectModal: (recipientId, referrerId, potId, potDetail) => {
     State.update({
       donateToProjectModal: { isOpen: true, recipientId, referrerId, potId, potDetail },
     });
   },
-  basisPointsToPercent: (basisPoints) => {
-    return basisPoints / 100;
-  },
   IPFS_BASE_URL,
```
[the remark this answers is not in the document]
Comment (louisdevzz) [reply apps/potlock/widget/Index.jsx:337]: I will updated

File: apps/potlock/widget/Index.jsx [hunk 4]
```diff
@@ -391,48 +291,50 @@ const props = {
...
+    return state.nearToUsd
+      ? "~$" + (amountNear * state.nearToUsd).toFixed(2)
+      : amountNear + " NEAR";
+  },
+  yoctosToUsdWithFallback: (amountYoctos) => {
+    return state.nearToUsd
+      ? "~$" + new Big(amountYoctos).mul(state.nearToUsd).div(1e24).toNumber().toFixed(2)
+      : new Big(amountYoctos).div(1e24).toNumber().toFixed(2) + " NEAR";
+  },
+  yoctosToUsd: (amountYoctos) => {
```
[the remark this answers is not in the document]
Comment (louisdevzz) [reply apps/potlock/widget/Index.jsx:327]: image
How to fix it?

[the remark this answers is not in the document]
Comment (louisdevzz) [reply apps/potlock/widget/Index.jsx:327]: It working but the error still occurs

File: apps/potlock/widget/Cart/BreakdownSummary.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (louisdevzz) [reply apps/potlock/widget/Cart/BreakdownSummary.jsx:1]: I was changed

**Review by elliotBraem — APPROVED** [review]

**Pull request merged**